app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -442,6 +442,24 @@ class StateFragmentTest {
     }
   }
 
+  @Test
+  @RunOn(TestPlatform.ESPRESSO) // Robolectric tests don't rotate like this to recreate activity
+  fun testStateFragment_loadExp_secondState_invalidAnswer_changeConfiguration_submitButtonIsEnabled() { // ktlint-disable max-line-length
```

Comment:
Can you think of a way to potentially reduce the length of this name, rather than disabling ktlint?

Reply:
E.g "configChange" in place of "changeConfiguration"